config/playground/docker/dev/scripts/custom_osd_entry.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
+
+import os
+import sys
+
+def run():
+    SOURCE_FILE = "/opt/opensearch-dashboards/extra_config/osd.yml"
```

Comment:
do we need to check if SOURCE_FILE existing ? Thanks

Reply:
Yeah, changed the structure to check for file existence in [new commit](https://github.com/opensearch-project/dashboards-anywhere/pull/91/commits/afa1c7d57264e293228d81b98676ddfc36f9cac4)